Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN \
         net-tools \
         nmap \
         openssh-client \
+        openssl \

```

Comment:
We don't need them at runtime for Home Assistant (which is what this repository is for)

You are welcome to add them and use our images as a base.